browser/ui/webui/brave_webui_source.cc

```diff
@@ -75,6 +75,7 @@ void CustomizeWebUIHTMLSource(const std::string &name, content::WebUIDataSource*
       }
     }, {
       std::string("welcome"), {
+      { "ecaefa14dcf65df518730a336430727e.ico", IDR_BRAVE_WELCOME_FAVICON },
```

Comment:
Just a question here - why are we using these hashes for filenames and not the actual readable (and permanent) name? Presumably webpack can be configured to keep them?

Reply:
only because `file-loader` defaults file names to md5 hash. I guess this came from the boilerplate which was carried over from b-l and just lacks custom config.

webpack handles this via file-loader `options` by declaring name `name: '[path][name].[ext]'`, but this seems to require changing all previous hashed files to their original names.